pkg/bridgeclient/app.go

```diff
@@ -72,6 +72,12 @@ func RunClient(test bool) {
 	if msgFormat == nil {
 		log.Fatalf("Unable to get the message format")
 	}
+
+	if pkg.Config.SkipTlsValidation {
+		log.Warn("SKIP_TLS_VALIDATION was set to true! Don't use this in production!")
+		bridgemodel.ConfigureDefaultTransportToSkipTlsValidation()
```

Comment:
are you sure the function name is descriptive enough???

Reply:
I really didn't want anyone to confuse it for something else 😅